server/src/test/Parsing.test.ts

```diff
@@ -88,9 +99,21 @@ function LibCube:Fact::getMeasureValue()
             );
             done();
         });
-    });
 
-    describe('function introduced with type Function', () => {
+        it('should parse correctly a function with a Function object as argument', (done) => {
```

Comment:
We say “parse correctly”, but strictly speaking we’re just checking the absence of errors and not the deduced meaning, right? But I think I asked you something very similar ages ago. :face_with_head_bandage: Can’t quite remember the answer.

Reply:
For this file, you're completely right. `should parse without errors` should be better.
There is another file where we check the resulting objects from the parsing.

Reply:
OK for _should parse without errors_.